tests/integration/config/jest.init.js

```diff
@@ -15,6 +16,12 @@ window.IntersectionObserver = jest.fn(() => ({
 
 Vue.use(ODS)
 
+Vue.use(GetTextPlugin, {
```

Comment:
Does this mean we can remove it from all the unit & integration tests?

Reply:
yes I will try to remove them and see if everything works